render-utils.js

```diff
@@ -0,0 +1,65 @@
+export function renderCharCard(char) {
+    const div = document.createElement('div');
+    const a = document.createElement('a');
+    const img = document.createElement('img');
+    const p = document.createElement('p');
+
+    div.classList.add('char-card');
+
+    p.textContent = char.name;
+    img.src = `./assets/${char.id}.JPG`;
+    a.href = `./details/?id=${char.id}`; 
+    
+    div.append(p, img);
+
+    a.append(div);
+
+    return a;
+}
+
+export function renderCharDetail(char) {
+    const div = document.createElement('div');
+    const img = document.createElement('img');
+    const descriptionEl = document.createElement('p');
+    const nameEl = document.createElement('p');
+    const ageEl = document.createElement('p');
+    const AffiliationEl = document.createElement('ul');
+    const AppearancesEl = document.createElement('ul');
+    const AffiliationAndAppearancesEl = document.createElement('div');
```

Comment:
I'm sure you'll notice how the syntax highlighting changes the color on your variables here on lines 26-28. I am not totally sure why(I have guesses) but I would just remind you to use `camelCase` for variables and save the upper casing on each word for special reserved words like `Math.random()` and `const data = new FormData(formEl)`